pkg/executor/cluster_table_test.go

```diff
@@ -188,6 +211,14 @@ func TestIssue20236(t *testing.T) {
 	srv := createRPCServer(t, dom)
 	defer srv.Stop()
 
+	// Mock cluster info to provide cluster topology
+	statusAddr := fmt.Sprintf("127.0.0.1:%d", config.GetGlobalConfig().Status.StatusPort)
+	instances := []string{
+		fmt.Sprintf("tidb,%s,%s,mock-version,mock-githash,1001", statusAddr, statusAddr),
+	}
+	fpExpr := `return("` + strings.Join(instances, ";") + `")`
+	testfailpoint.Enable(t, "github.com/pingcap/tidb/pkg/infoschema/mockClusterInfo", fpExpr)
```

Comment:
is this new failpoint related to the flaky?

Reply:
It is used for debug, removed